### DISCUSSION TITLE
How goldfish help me to protect my Server?

### DISCUSSION THREAD

**Discussion by giscus** [thangchiba/ThangChibaBlog#5] [Announcements]
# How goldfish help me to protect my Server?

Ever wondered if computers can truly generate random numbers? How could seemingly clueless goldfish help in concocting random codes to thwart hackers? This is akin to what CloudFlare is doing to safeguard a huge chunk of the world's internet traffic.

https://blog.thangchiba.com/blog/random-generator/

**Comment by thangchiba** [comment]
Cá vàng
